lcd_test/testdata/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+PASSWORD="1234567890"
+ADDR="cosmos16xyempempp92x9hyzz9wrgf94r6j9h5f06pxxv"
+RECEIVER="cosmos17gx5vwpm0y2k59tw0x00ccug234n56cgltx2w2"
+VALIDATOR="cosmosvaloper16xyempempp92x9hyzz9wrgf94r6j9h5f2w4n2l"
+AMOUNT="1000000stake"
+CHAIN="lcd"
+GOVID="2"
+HOME="/tmp/contract_tests/.gaiacli"
+SWAGGER='/tmp/contract_tests/swagger.yaml'
+
+# luckily governance are down in the swagger sequence of calls, this 15s are massive sleep time
+# TODO: find out why the signature verification still fails without sleeps
+# 3 seconds works sometims, 4 seconds often, 5 always but is huge!
```

Comment:
ditto,  I only see `1 s` sleeps

Reply:
I reduced the precommit_timeout from 5s to 500ms, I'll fix the comment here